test/components/popup.spec.tsx

```diff
@@ -28,68 +50,82 @@ describe('<Popup />', function() {
 
     describe('The popup user', function() {
         it('clicks on the parent and the popup opens and closes after another click', async function() {
-            const {select, waitForDom} = clientRenderer.render(<PopupDemo />);
+            const {result, waitForDom} = clientRenderer.render(<PopupDemo />);
+
+            let popupDemo = new PopupDemoTestDriver(result as PopupDemo);
 
             await waitForDom(() => {
-                expect(select(demoContainer)).to.be.present();
-                expect(select(demoContainer, portalId)).to.be.absent();
+                expect(popupDemo.container).to.be.present();
+                expect(popupDemo.isPopupPresent).to.be.false;
```

Comment:
please convert to  `to.equal(false)` everywhere

Reply:
fixed